src/python/ensembl/io/genomio/integrity.py

```diff
@@ -716,30 +713,19 @@ def _compare_seqs(
         return comp
 
 
-class InputSchema(argschema.ArgSchema):
-    """Input arguments expected by this script."""
-
-    # Server parameters
-    manifest_file = argschema.fields.InputFile(
-        required=True, metadata={"description": "Manifest file for the data to check"}
-    )
-    brc_mode = argschema.fields.Boolean(required=False, metadata={"description": "BRC mode"})
-    ignore_final_stops = argschema.fields.Boolean(
-        required=False, metadata={"description": "Ignore final stop when calculating peptide length"}
-    )
-
-
 def main() -> None:
     """Main entrypoint."""
-    mod = argschema.ArgSchemaParser(schema_type=InputSchema)
-
-    # Start
-    inspector = IntegrityTool(mod.args["manifest_file"])
-    if mod.args.get("brc_mode"):
-        inspector.set_brc_mode(True)
-    if mod.args.get("ignore_final_stops"):
-        inspector.set_ignore_final_stops(True)
+    parser = ArgumentParser(
+        description="Compare the genomic data between the files present in a manifest file."
+    )
+    parser.add_argument_src_path("--manifest_file", required=True, help="Manifest file for the data to check")
+    parser.add_argument("--brc_mode", action="store_true", help="Enable BRC mode")
```

Comment:
The nextflow process needs to be updated to provide only --brc_mode instead of --brc_mode 1

Reply:
I'm waiting to rebase this branch following PRs #173 and #180 before doing such changes.